include/revng-c/TypeNames/ModelToPTMLTypeHelpers.h

```diff
@@ -88,58 +88,58 @@ extern void printForwardDeclaration(const model::TypeDefinition &T,
                                     ptml::PTMLIndentedOstream &Header,
                                     ptml::PTMLCBuilder &B);
 
-// Print a declaration for a Type. The last three arguments (`TypesToInline`
-// `NameOfInlineInstance` and `Qualifiers`) are being used for printing types
-// inline. If the `NameOfInlineInstance` and `Qualifiers` are set, it means that
-// we should print the type inline. Types that can be inline are structs, unions
-// and enums.
+// Print a declaration corresponding to a given `model::TypeDefinition`.
 extern void
 printDeclaration(Logger<> &Log,
                  const model::TypeDefinition &T,
                  ptml::PTMLIndentedOstream &Header,
                  ptml::PTMLCBuilder &B,
                  const model::Binary &Model,
-                 std::map<model::QualifiedType, std::string> &AdditionalNames,
-                 const std::set<const model::TypeDefinition *> &TypesToInline,
-                 llvm::StringRef NameOfInlineInstance = llvm::StringRef(),
-                 const std::vector<model::Qualifier> &Qualifiers = {},
-                 bool ForEditing = false);
```

Comment:
Can you briefly summarize why we no longer need any of these arguments?
Even `ForEditing`? Seems weird.

Reply:
> Because none of them was ever read

Ah. Well, no further explanation needed `:D`